packages/terra-collapsible-menu-view/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ ChangeLog
 
 Unreleased
 ----------
+### Updated
+* Removed `large` viewport from terra-collapsible-menu-view 
```

Comment:
Lets update this to reflect the recent PR changes. We should be able to add these changes under the `### Changed` heading. Here's more info on the changelog semantics we follow: https://keepachangelog.com/en/1.0.0/